libbeat/docs/dashboards.asciidoc

```diff
@@ -102,6 +102,10 @@ predefined +{beatname_lc}-*+ index pattern and set it as the default.
 
 image:./images/kibana-created-indexes.png[Kibana configured indexes]
 
+NOTE: After importing the index-pattern in Kibana, you need to refresh each index-pattern in Kibana in order to update
```

Comment:
@dedemorton yes. I am referring to the indices refresh from Kibana. As suggested I also added to FAQ.